app/assets/javascripts/discourse/app/widgets/post-menu.js

```diff
@@ -12,25 +12,30 @@ const VIBRATE_DURATION = 5;
 const _builders = {};
 const _extraButtons = {};
 export let apiExtraButtons = {};
+let _buttonsToRemove = {};
 
 export function addButton(name, builder) {
   _extraButtons[name] = builder;
 }
 
 export function resetPostMenuExtraButtons() {
   Object.keys(apiExtraButtons).forEach((button) => {
-    removeButton(button);
+    if (_extraButtons[button]) {
+      delete _extraButtons[button];
+    }
   });
```

Comment:
Do we need that loop to reset? Can we just set `_extraButtons = {};` ?

Reply:
You're right here, we can just reset the object.